.github/workflows/build.yml

```diff
@@ -0,0 +1,84 @@
+name: Build and Push to Dockerhub
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```

Comment:
I'm surprised this is needed

Reply:
Yeah, me too until I realised [github does not have arm64 runners](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners), so we still have to build the arm image on a x86 setup, which relies on qemu.